src/storage/sqlite.py

```diff
@@ -1,45 +1,59 @@
-import collections
-from threading import current_thread, enumerate as threadingEnumerate, RLock
-import Queue
+"""
+src/storage/sqlite.py
+=====================
+"""
+
 import sqlite3
 import time
+from threading import RLock
+
+from helper_sql import sqlQuery, sqlExecute, SqlBulkExecute
+from storage import InventoryItem, InventoryStorage
 
-from helper_sql import *
-from storage import InventoryStorage, InventoryItem
 
 class SqliteInventory(InventoryStorage):
+    """"""
     def __init__(self):
-        super(self.__class__, self).__init__()
-        self._inventory = {} #of objects (like msg payloads and pubkey payloads) Does not include protocol headers (the first 24 bytes of each packet).
-        self._objects = {} # cache for existing objects, used for quick lookups if we have an object. This is used for example whenever we receive an inv message from a peer to check to see what items are new to us. We don't delete things out of it; instead, the singleCleaner thread clears and refills it.
-        self.lock = RLock() # Guarantees that two receiveDataThreads don't receive and process the same message concurrently (probably sent by a malicious individual)
+        super(SqliteInventory, self).__init__()
+        # of objects (like msg payloads and pubkey payloads) Does not include
+        # protocol headers (the first 24 bytes of each packet).
+        self._inventory = {}
+        self._objects = {}
+        # cache for existing objects, used for quick lookups if we have an object. This is used for example whenever
```

Comment:
This comment should be moved one line up

Reply:
Github is highlighting two comments there, I'm guessing you're after a
```
# comment
ass = ignment
```
pattern so I've applied that consistently in the `__init__` method.

Reply:
Swap lines 21 and 22.